package.json

```diff
@@ -82,6 +82,7 @@
     "@trivago/prettier-plugin-sort-imports": "^4.2.0",
     "@types/benchmark": "^2.1.4",
     "@types/chalk": "^2.2.0",
+    "@types/debug": "^4.1.12",
```

Comment:
Are we using the [`debug`](https://github.com/debug-js/debug) package? :raised_eyebrow: